frontend/src/views/applicants/import-export.tsx

```diff
@@ -18,6 +18,48 @@ import {
     ApplicantsDiffList,
 } from "../../components/applicants";
 
+/**
+ * Converts a list of applicants into a File object
```

Comment:
That's not what it does...

Reply:
Changed to comment from original code on master branch